src/enums/Heroes.ts

```diff
@@ -0,0 +1,34 @@
+export enum HeroIndex {
+    Human = 0,
+    Orc = 1,
+    NightElf = 2,
+    Undead = 3,
+    Tavern = 4
+}
+
+export enum HeroNames {
```

Comment:
i would actually prefer having this in the backend as an auto property. You can calcualte this in the readmodel aswell.